<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParentWidget</class>
 <widget class="QWidget" name="ParentWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>608</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="4">
    <widget class="QPushButton" name="continueButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Continue</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QTabWidget" name="parentTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="runParameterTab">
      <attribute name="title">
       <string>Run Parameters</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="runStagesTab">
      <attribute name="title">
       <string>Run Stages</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="backButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
